build.gradle

```diff
@@ -48,6 +48,6 @@ subprojects {
 }
 
 javaVersions {
-    libraryTarget = 11
+    libraryTarget = 17
```

Comment:
we likely won't be able to upgrade this for a couple weeks in a few places, but that's fine